content/spin/v2/triggers.md

```diff
@@ -13,9 +13,70 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/triggers.md"
 
 A Spin _trigger_ maps an event, such as an HTTP request or a Redis pub-sub message, to a component that handles that event.
 
-An application can contain multiple triggers, but they must all be the same type. For example, an application can contain triggers for multiple HTTP routes, or for multiple Redis pub-sub channels, but can't contain both an HTTP _and_ a Redis trigger.
+An application can contain multiple triggers. As of Spin 2.3 onward, a Spin application can use any combination of trigger types within a single application. For example, a single application can contain triggers for both HTTP routes and Redis pub-sub channels. 
+
+> Currently, only the HTTP template supports the `spin add` command. If you wish to use `spin add` to build an application with multiple triggers, **including a Redis trigger**, you should begin with the Redis trigger application. Then, `spin add` your HTTP trigger component(s) as demonstrated below. Please note that additional Redis trigger components cannot be added via `spin add` at this time.
+
+Here is a quick example of creating an application with both HTTP and Redis triggers:
+
+<!-- @nocpy -->
+
+```bash
+# Start with a Redis trigger application
+$ spin new -t redis-rust trigger-example
+Description: A multiple trigger example
+Redis address: redis://localhost:6379
+Redis channel: one
+# Change into to the application directory
+$ cd trigger-example 
+# Create HTTP trigger application
+$ spin add -t http-rust http-trigger-example  
+Description: A HTTP trigger example
+HTTP path: /...
+```
+
+The above `spin new` and `spin add` commands will automatically scaffold the following Spin manifest (`spin.toml` file):
+
+<!-- @nocpy -->
+
+```toml
+spin_manifest_version = 2
+
+[application]
+name = "trigger-example"
+version = "0.1.0"
+authors = ["Fermyon Engineering <engineering@fermyon.com>"]
+description = "A multiple trigger example"
+
+[application.trigger.redis]
+address = "redis://localhost:6379"
+
+[[trigger.redis]]
+channel = "one"
+component = "trigger-example"
+
+[component.trigger-example]
+source = "target/wasm32-wasi/release/trigger_example.wasm"
+allowed_outbound_hosts = []
+[component.trigger-example.build]
+command = "cargo build --target wasm32-wasi --release"
+
+[[trigger.http]]
+route = "/..."
+component = "http-trigger-example"
+
+[component.http-trigger-example]
+source = "http-trigger-example/target/wasm32-wasi/release/http_trigger_example.wasm"
+allowed_outbound_hosts = []
+[component.http-trigger-example.build]
+command = "cargo build --target wasm32-wasi --release"
+workdir = "http-trigger-example"
+watch = ["src/**/*.rs", "Cargo.toml"]
+```
+
+> If you're familiar with [Spin 1.x](/spin/manifest-reference-v1#the-trigger-table), note that [Spin 2 uses the term "trigger"](/spin/manifest-reference#the-trigger-table) to refer to each individual route or channel, rather than the trigger type. It's closer to the `[component.trigger]` usage than to the application trigger.
```

Comment:
This shouldn't be in the middle of a discussion of multi-triggering - it's general orientation material for people migrating from 1.x.

Reply:
Move back to the original order in latest push.